services/clients/agent/pulse2agent/inventory.py

```diff
@@ -178,7 +178,7 @@ def posix_shell_query(self, base_command, software_required):
         @rtype: generator
         """
 
-        if len(software_required) > 1:
```

Comment:
je pense qu'ici il faut laisser le len comme il faut > strict a 1

Reply:
Do not change this one